packages/template-retail-react-app/app/page-designer/README.md

```diff
@@ -1,31 +1,31 @@
-        ____                      ____            _                      
+        ____                      ____            _
        / __ \____ _____ ____     / __ \___  _____(_)___ _____  ___  _____
       / /_/ / __ `/ __ `/ _ \   / / / / _ \/ ___/ / __ `/ __ \/ _ \/ ___/
-     / ____/ /_/ / /_/ /  __/  / /_/ /  __(__  ) / /_/ / / / /  __/ /    
-    /_/    \__,_/\__, /\___/  /_____/\___/____/_/\__, /_/ /_/\___/_/     
-                /____/                          /____/                   
+     / ____/ /_/ / /_/ /  __/  / /_/ /  __(__  ) / /_/ / / / /  __/ /
+    /_/    \__,_/\__, /\___/  /_____/\___/____/_/\__, /_/ /_/\___/_/
+                /____/                          /____/
 
 ---
 
-## Summary
+This folder contains React components and utilities that render pages from [Page Designer](https://documentation.b2c.commercecloud.salesforce.com/DOC2/topic/com.demandware.dochelp/content/b2c_commerce/topics/page_designer/b2c_creating_pd_pages.html).
 
-This folder contains the React components and utilities required to render `pages` created in the Business Managers **Page Desginer** backend in your PWA-Kit app. This is a living folder where you, the developer, can add your own component implementations for those that exist in the **Page Designer** backend, or custom components you may have added via an SFRA cartirdge. We have given you a head start by providing some of the components for layout and visualization (e.g. images, grids, and carousels). 
+It is living folder where you add React.js components responsible for rendering Page Designer components that have been serialized to JSON.
```

Comment:
Did you mean to say "It is a living" .. 

Is "It" page designer? or the page designer folder?

Reply:
I don't think "living folder" translates well to other cultures.

Reply:
I agree. But I don't have any suggestions for alternative 🤷. We could be more direct and say something along the lines of "this is where you'll place all your page designer components that you develop"